cv32/bsp/Makefile

```diff
@@ -1,12 +1,12 @@
 RISCV            ?= /opt/riscv
-RISCV_EXE_PREFIX  = $(RISCV)/bin/riscv32-unknown-elf-
+RISCV_EXE_PREFIX ?= $(RISCV)/bin/riscv32-unknown-elf-
 RISCV_GCC = $(RISCV_EXE_PREFIX)gcc
 RISCV_AR = $(RISCV_EXE_PREFIX)ar
 SRC = crt0.S handlers.S syscalls.c vectors.S
 OBJ = libcv-verif.a crt0.o handlers.o syscalls.o vectors.o
 
 all:
-	$(RISCV_GCC) -Os -g -c -static $(SRC)
+	$(RISCV_GCC) -Os -g -c -static -mabi=ilp32 -march=rv32imc $(SRC)
```

Comment:
I am very reluctant to accept this change for two reasons:
1. The author of the BSP is @craigblackmore and he has much more extensive knowledge of toolchain and BSP issues than I do.
2. It seems to me that this change restricts the machine to IMC instructions.   That is OK for now, but in a few weeks we will need to support Z and Xpulp instructions.

Reply:
Is this code relying on Z and Xpulp instructions?
The goal of this modification is to make explicit the ABI and the architecture as done in Common.mk

Reply:
I would suggest defining common options in a CFLAGS variable in Common.mk e.g.
```
CFLAGS = -Os -g -static -mabi=ilp32 -march=rv32imc -Wall -pedantic
```
then the make targets can use `$(CFLAGS)` rather than hard-coding the flags in multiple places.

In bsp/Makefile we can do:
```
CFLAGS ?= -Os -g -static -mabi=ilp32 -march=rv32imc -Wall -pedantic
```
and again use `$(CFLAGS)` instead of hard-coding the flags.

Reply:
OK

Reply:
A new version taking into account @craigblackmore suggestion is available.